### PR TITLE
Add DateTime helpers unittests

### PR DESCRIPTION
**Pull request by lookacat**
## Description
Add unittests for datetime helper functions

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [x] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: packages/web-pkg/tests/unit/helpers/datetime.spec.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      [
+        DateTime.fromISO('2010-10-22T21:38:00'),
+        'en',
+        DateTime.DATETIME_MED,
+        'Oct 22, 2010, 9:38 PM'
+      ],
+      [DateTime.fromISO('2010-10-22T21:38:00'), 'fr', DateTime.DATETIME_MED, '22 oct. 2010, 21:38'],
+      [DateTime.fromISO('2010-10-22T21:38:00'), 'it', DateTime.DATETIME_MED, '22 ott 2010, 21:38']
+    ])(
+      'invalid input "%s"',
```
[the remark this answers is not in the document]
Comment (lookacat) [reply packages/web-pkg/tests/unit/helpers/datetime.spec.ts:35]: oh oops forgot to change it /.- srry

[the remark this answers is not in the document]
Comment (lookacat) [reply packages/web-pkg/tests/unit/helpers/datetime.spec.ts]: changed it

**Review by JammingBen — APPROVED** [review]

File: packages/web-pkg/tests/unit/helpers/datetime.spec.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  formatDateFromDateTime,
+  formatDateFromJSDate,
+  formatDateFromHTTP,
+  formatDateFromISO,
+  formatDateFromRFC,
+  formatRelativeDateFromDateTime,
+  formatRelativeDateFromJSDate,
+  formatRelativeDateFromISO,
+  formatRelativeDateFromRFC
+} from '../../../src/helpers'
```
Comment (kulmann) [inline packages/web-pkg/tests/unit/helpers/datetime.spec.ts:12]:
```suggestion
} from 'web-pkg'
```
`web-pkg` exports all its members, so you can just import from `web-pkg`.

**Review by kulmann — CHANGES_REQUESTED** [review]
I have some more requests for this 😅 

1. please don't vendor-test different languages. We need to assume that the lib does what it's supposed to do for the different languages. this means you can hard code the language to one that you like (e.g. `en`) / keep it in a const / whatever you like, but it should not be test input. Same applies for the datetime format - we must assume that the lib handles its own datetime formats correctly, so you should remove them from the test input again and then only add the format in a hardcoded way where you call the respective function.
2. unit tests are supposed to not only test the happy path but also check error handling. Ideally each function from tested file should have a `describe` block (that's already the case) and then have a positive test (valid input leads to valid output) and a negative test (invalid input leads to some exception, a string that indicates invalid input, something alike). If there are more edge cases it's a good idea to test them as well. To be clear: the scope of tests for the date time conversion is, to test that the various conversion functions convert a date/time of a certain format (iso string, js object, ...) to a string representation.
3. Please be as strict with the types as possible. I saw some occurrences where types were not missing or `any` was given.

File: packages/web-pkg/tests/unit/helpers/datetime.spec.ts (hunk 2 above)
Comment (dschmidt) [reply packages/web-pkg/tests/unit/helpers/datetime.spec.ts]: Although it might be that vite doesn't like this and we'll have to revert it in the vite PR :trollface:

**Comment by lookacat** [comment]
done

**Review by kulmann — COMMENTED** [review]
I don't understand why you have the additional boilerplate of `it.each` tests but only ever have one test example. Are you planning on adding more test examples? If not please refactor to single test statements.

**Comment by lookacat** [comment]
no you are right, its better that way. done

**Review by kulmann — APPROVED** [review]
🥳
